lib/ProductOpener/Config_off.pm

```diff
@@ -188,6 +188,7 @@ $flavor = 'off';
 	stephane
 	tacinte
 	teolemon
+	tchen1
```

Comment:
You must not commit that !

Reply:
Sorry! I will get on the fixes hopefully this weekend or tuesday